catt/stream_info.py

```diff
@@ -7,27 +7,27 @@
 from .util import get_local_ip, guess_mime
 
 AUDIO_DEVICE_TYPES = ["audio", "group"]
-ULTRA_MODELS = [("Xiaomi", "MIBOX3"), ("Google Inc.", "Chromecast Ultra")]
+ULTRA_MODELS = [("Xiaomi", "MIBOX3"), ("Unknown manufacturer", "Chromecast Ultra")]
```

Comment:
The **manufacturer** field is only available though DIAL, so this is now completely unavailable for Google devices.

Reply:
Should this be a comment in the code?

Reply:
It was FYI, but a comment won't hurt. :smiley: